sass/woocommerce/_archives.scss

```diff
@@ -83,7 +83,7 @@
 				text-align: center;
 
 				@media (max-width: 768px) {
-					& {
+					@at-root body.responsive#{&} {
```

Comment:
I'll try switching the the SASS compiler that comes with PHPStorm. It's possible the one that comes with LiveReload is too old.